src/Symfony/Component/Messenger/Middleware/ValidationMiddleware.php

```diff
@@ -28,6 +29,9 @@ public function __construct(ValidatorInterface $validator)
         $this->validator = $validator;
     }
 
+    /**
+     * @param Envelope $envelope
+     */
```

Comment:
should be backported to 4.1

Reply:
I don't think we need to backport this, not worth it.